wordpress-mobile/WordPress-iOS
# [Media Editing] Load images from Private sites

Fixes #13327

### To test

1. Go to Settings -> Privacy and change one of your sites to "Private"
2. Create a post using the block editor then add an image
3. Tap the block -> Tap the edit icon -> Tap "Edit"
4. Check that the image loads

If you test the flow above in `develop` the Media Editor will display an error message.

PR submission checklist:
